auto_nag/scripts/configs/tools.json

```diff
@@ -132,7 +132,7 @@
     "unassign_weeks": 2,
     "number_of_months": 7,
     "max_ni": 7,
-    "max_unassign_per_triage_owner": 7,
+    "max_actions": 10,
```

Comment:
Why 10?

Reply:
The `max_actions` already include the `max_ni`. For example, if a triage owner ended up with 20 needinfos and 35 auto-fixs. We will put 7 out of the 20 needinfos and 3 out of the 35 auto-fixs.

I selected **10** to have a similar number of bugs to what we already have with the current thresholds.

Reply:
Let's use `14` then, so we allow a maximum of 7 needinfos and a maximum of 7 autofixes without ni.